explore/src/main/scala/explore/targeteditor/GmosGeometry.scala

```diff
@@ -64,12 +64,14 @@ object GmosGeometry:
     configuration: Option[BasicConfiguration],
     port:          PortDisposition,
     extraCss:      Css = Css.Empty
-  ): (Css, ShapeExpression) =
-    (Css("patrol-field-intersection") |+| extraCss) ->
-      offsets
-        .map(patrolField(posAngle, _, configuration, port))
-        .collect { case Some(s) => s }
-        .reduce(_ ∩ _)
+  ): Option[(Css, ShapeExpression)] =
+    configuration.map(_ =>
+      (Css("patrol-field-intersection") |+| extraCss) ->
+        offsets
+          .map(patrolField(posAngle, _, configuration, port))
```

Comment:
`patrolField` is not used anywhere else, I think. So it could be updated to accept a `BasicConfiguration` instead of `Option[BasicConfiguration]`

Reply:
Right, we could do that